lib/tasks/lang.rake

```diff
@@ -78,7 +78,7 @@ namespace :lang do
   task setup: [:environment, :login, :verbose, :safe_mode]
 
   desc "Log in user for import tasks."
-  task(:login) do
+  task(login: :environment) do
```

Comment:
@mo-nathan @pellaea 
I added `:environment` as a dependency to 4 tasks.
- Question: Are these tasks ever used? 

- I don't understand the tasks.
But without this dependency, I got
```ruby
joe@iMac mushroom-observer % rake lang:login_admin
rake aborted!
NameError: uninitialized constant User (NameError)

    User.current = User.find(0)
    ^^^^
/Users/joe/mushroom-observer/lib/tasks/lang.rake:91:in `block (2 levels) in <top (required)>'
Tasks: TOP => lang:login_admin
(See full trace by running task with --trace)
```

Reply:
I for sure don't intentionally use them.  However, they might get called indirectly.  In my view our translation stuff is kinda of wonky so I would move carefully.